go.mod

```diff
@@ -3,8 +3,12 @@ module github.com/cmsgov/easi-app
 go 1.13
 
 require (
+	github.com/graph-gophers/graphql-go v0.0.0-20191115155744-f33e81362277 // indirect
```

Comment:
Any idea where this is coming from? Feel like graphql should be getting imported from any of the packages we're using, but it's possible.

Reply:
It's very likely from me when I was exploring whether or not we should use graphql and snuck in. We don't need it though. I've built without it.